packages/nimble-components/package.json

```diff
@@ -101,7 +103,7 @@
     "rollup-plugin-terser": "^7.0.2",
     "storybook-addon-xd-designs": "^6.0.0",
     "ts-loader": "^9.2.5",
-    "typescript": "^4.3.2",
+    "typescript": "~4.3.2",
```

Comment:
Can we not update the typescript version to align with angular? The fast-element-1 branch we use releases from should be compatible: https://github.com/microsoft/fast/blob/archives/fast-element-1/packages/web-components/fast-foundation/package.json#L94

IS there some other issue that prevents aligning on ~4.6.4 across packages?

Reply:
I did not include a decision to upgrade the components and tokens as part of this PR. I will test upgrading these, but if there are issues then I will continue as is.